tests/integration/testcli.nim

```diff
@@ -9,32 +9,14 @@ suite "Command line interface":
   let key = "4242424242424242424242424242424242424242424242424242424242424242"
 
   test "complains when persistence is enabled without ethereum account":
-    let node = startNode(@["--persistence"])
+    let node = startNode(@["persistence"])
     node.waitUntilOutput("Persistence enabled, but no Ethereum account was set")
     node.stop()
 
-  test "complains when validator is enabled without ethereum account":
```

Comment:
Are we sure we don't want to give these warnings to the user? They seem like config mistakes that are easily made.